src/pages/home/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useAppDispatch, useAppSelector } from '../../app/hooks'
+import {increment, decrement, incrementByAmount } from '../../features/counter/counterSlice'
```

Comment:
Styling issue with the space, lets try to add the project linting as early as possible in another PR
```suggestion
import { increment, decrement, incrementByAmount } from '../../features/counter/counterSlice'
```